src/build/patches.ts

```diff
@@ -92,11 +92,15 @@ function handleMixin(node: Node): DeepPartial<Interface> {
     }
   }
 
-  return {
+  const result: any = {
     name,
     events: { event },
     properties: { property },
-  } as DeepPartial<Interface>;
+  };
```

Comment:
```suggestion
  const result = {
name,
events: { event },
properties: { property },
  } as DeepPartial<Interface>;
```

It's not clear to me why `const result: DeepPartial<Interface>` doesn't work (the error is confusingly complex), but this should be ok.